webui.py

```diff
@@ -3148,12 +3148,57 @@ def doc_upload(self):
         except Exception, e:
             raise FormError, "Error unpacking zipfile:" + str(e)
 
-        self.store.set_has_docs(name)
-        self.store.log_docs(name, version)
+        self.store.set_has_docs(name, value=True)
+        self.store.log_docs(name, version, 'docupdate')
         self.store.changed()
         raise Redirect("https://pythonhosted.org/%s/" % name)
 
     #
+    # Documentation Destruction
+    #
+    @must_tls
+    def doc_destroy(self):
```

Comment:
Should this be added to https://github.com/pypa/pypi-legacy/blob/7162272d49edbe8c99c021af7fb6b99cab8a42d5/webui.py#L857 Sorry, for some reason I can't get my local setup work at the moment so I can't test the PR :(